collectors/property.yaml

```diff
@@ -22,8 +22,24 @@ HostSystemPropertiesCollector:
       - label: "runtime_memory_capacity_kilobytes"
         property: "runtime|memoryCap"
 
+VMPropertiesCollector:
+    enum_metrics:
+      - label: "runtime_powerState"
+        expected: "Powered On"
+        property: "summary|runtime|powerState"
+      - label: "runtime_connectionState"
+        expected: "connected"
+        property: "summary|runtime|connectionState"
+      - label: "summary_numEthernetCards"
+        expected: "1"
+        property: "summary|config|numEthernetCards"
```

Comment:
It's more like a number metric.

Reply:
I tested it in number metrics as well but it dont return the value.  It was add in Info_metrics first but that also didnt helping out.  I commented the code for info_metrics in the collector but then testing was failing. then i added "summary_parentVcenter" in info_metrics and moved "summary_numEthernetCards" to enum metrics.

Reply:
Make it a number and figure out, why it isn't working. We can discuss it in the channel too, if you have the exact error message.

Reply:
In the meantime I have tested the metrics on my computer and constantly get back numerical values. Let me know if you need any more help.